articles/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Article(models.Model):
+    DoesNotExist = None
```

Comment:
what is the reason to create this attribute? Django provides your default exception `DoesNotExist`. So, I believe we could remove this attribute. Feel free to correct me if I am wrong

Reply:
Yes, can remove.
The reason was that pycharm suggested to put it, when I've created ArticleForm. 
If understand correctly can use ArticleCreateForm or ArticleForm in app.